doc/advanced_configuration.rst

```diff
@@ -5,24 +5,10 @@ Advanced Configuration
 Here are the personal configurations that you can modify within Sphinx-Gallery.
 
 
-Changing default directories
-============================
```

Comment:
Is not really removed.  It's only moved to the main readme. Making part of the standard configuration to declare where your examples directories are.